service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantAboutToStartControllerTest.java

```diff
@@ -45,17 +46,17 @@
 
     @BeforeEach
     void setup() {
-        given(organisationService.findOrganisation()).willReturn(POPULATED_ORGANISATION);
+        given(organisationService.findOrganisation()).willReturn(Optional.of(POPULATED_ORGANISATION));
         given(authTokenGenerator.generate()).willReturn(SERVICE_AUTH_TOKEN);
-        given(organisationApi.findOrganisationById(USER_AUTH_TOKEN, SERVICE_AUTH_TOKEN))
+        given(organisationApi.findUserOrganisation(USER_AUTH_TOKEN, SERVICE_AUTH_TOKEN))
             .willReturn(POPULATED_ORGANISATION);
     }
 
     @Test
     void shouldPrepopulateApplicantDataWhenNoApplicantExists() {
```

Comment:
It is slightly off topic for your story but do you think we should actually assert the contents of the built collection (that is is actually empty) instead of just that there is an entry for it in the map?